src/ml_board/frontend/Dashboard/src/worker_socket/event_handlers/evaluationResultDataHandler.tsx

```diff
@@ -1,5 +1,5 @@
 export interface evalResultCustomData {
-    grid_search_id: string, //TODO: move it to the statusSlice?
+    grid_search_id: string | null, //TODO: move it to the statusSlice?
```

Comment:
who do we allow `null` here?

Reply:
I think you mean why? well because in **MainThreadCallbacks.ts** line 12, it is initialize with null.

Honestly this is temporary and the whole file (and the initialization mentioned) will probably see some changes soon!

Reply:
alright then :+1: